sandy/fy.py

```diff
@@ -390,7 +390,7 @@ def apply_bmatrix(self, zam, e, decay_data):
         new_data.loc[mask, 'FY'] = fy_calc.values
         return self.__class__(new_data)
 
-    def apply_qmatrix(self, zam, energy, decay_data):
+    def apply_qmatrix(self, zam, energy, decay_data, reindex=False):
```

Comment:
I don't like calling it `reindex`..it's not fully clear what it does.
Maybe something like `keep_ify_index=False`